packages/vapor/scss/forms/block-form.scss

```diff
@@ -70,17 +70,22 @@ form {
             }
 
             > header:not(:first-child) {
-                margin-top: $form-group-margin;
+                margin-top: $spacing;
             }
 
             > *:not(:first-child) {
                 margin-top: $spacing * 2;
                 &.input-field {
                     margin-top: $form-group-margin + $input-margin-top;
                 }
-                &.form-group {
-                    margin-top: $form-group-margin;
-                }
+            }
+
+            > *:not(:first-of-type).form-group {

```

Comment:
Those changes break the default spacing for the Sections

![image](https://user-images.githubusercontent.com/8355585/67390166-18a02e00-f56a-11e9-9099-ed959ad79225.png)

I think we need to see why you needed to change that for your use case and see whether we need to add something here or change the layout in your Examples

Reply:
Indeed before merging I would like us three [Chuck, you, me] to meet and check if we keep the CSS as it is and modify my layout or go with Chuck Idea to wrap everything in Section(s)